src/helpers/documentHelper.js

```diff
@@ -3,35 +3,7 @@
  * See LICENSE in the project root for license information.
  */
 
-/* global Excel, Office, Word, Promise */
```

Comment:
Why are these file changes needed for the JS case and they were not needed for the ts?

Reply:
The js case was structured slightly differently from the ts case.  I did some modifications to bring it more in line with the ts template (like modifying this file which doesn't exist in the ts case), but didn't line it up completely.

Reply:
Specifically . . . I left a method here that was shared with the writeDataTo* methods instead of copying it's functionality to each of the different host methods like it is in ts.